filesmanager/processors/xapi/event_transformers/filesmanager_events.py

```diff
@@ -0,0 +1,39 @@
+"""
+Transformers for filesmanager events.
+
+Classes:
+    FilesDownloadedTransformer: Transformer for the event edunext.xblock.filesmanager.files.downloaded.
+"""
+
+from tincan import Activity, ActivityDefinition, LanguageMap, Verb
+
+from filesmanager.edxapp_wrapper.event_routing_backends import XApiTransformer, XApiTransformersRegistry
+from filesmanager.processors.xapi import constants
+
+
+@XApiTransformersRegistry.register("edunext.xblock.filesmanager.files.downloaded")
+class FilesDownloadedTransformer(XApiTransformer):
+    """
+    Transformers for event generated when an student download files from xblock.
+    """
+
+    _verb = Verb(
+        id=constants.XAPI_VERB_DOWNLOADED,
+        display=LanguageMap({constants.EN: constants.DOWNLOADED}),
+    )
+
+    additional_fields = ("result",)
```

Comment:
if you add this you have to implement the method get_result

Reply:
Do you think it is useful to use this method in this case, or is it better to remove the additional fields?

Reply:
that depends on your requirements, but probably this is not necessary

Reply:
I'll remove the attribute